util/Directories.cpp

```diff
@@ -734,3 +734,25 @@ auto IsExistingFile(const fs::path& path) -> bool
 
     return false;
 }
+
+auto ReadFOCScript(boost::filesystem::path const& path, std::string& file_contents) -> bool
```

Comment:
this isn't FOCS specific... the old name was more accurate

Reply:
OK, changed to `ReadFile`.